tgui/packages/tgui/interfaces/HighLuminosityEyesMenu.tsx

```diff
@@ -0,0 +1,171 @@
+            step={1}
+            stepPixelSize={5}
+            value={range}
+            minValue={0}
+            maxValue={5}
+            onDrag={(e, value) =>
+              act('set_range', {
+                new_range: value,
+              })
+            }
+          />
+        </LabeledList.Item>
+      </LabeledList>
+    </Section>
+  );
+};
+
+const EyeColorDisplay = (props, context) => {
+  const { act, data } = useBackend<HighLuminosityEyesMenuData>(context);
+  const { eyeColor } = data;
+  const left = 1;
+  const right = 2;
+  return (
+    <Section title="Eye Color">
+      <LabeledList>
+        <LabeledList.Item label="Matches Light Color">
+          <Button.Checkbox
+            checked={eyeColor.mode}
+            onClick={() => act('toggle_eye_color')}
+            tooltip="Toggle the eye color mode."
+          />
+        </LabeledList.Item>
+      </LabeledList>
+      {!eyeColor.mode && (
+        <LabeledList>
+          &nbsp;
+          <LabeledList.Item label="Left Eye">
+            <ColorBox color={eyeColor.left} />{' '}
+            <Button
+              icon="palette"
+              onClick={() => act('pick_color', { to_update: left })}
+              tooltip="Brings up a color pick window to change the light color."
+            />
+            <Button
+              icon="dice"
+              onClick={() => act('random_color', { to_update: left })}
+              tooltip="Randomizes the light color."
+            />
+            <Input
+              value={eyeColor.left}
+              width={6}
+              maxLength={7}
+              onChange={(_, value) =>
+                act('enter_color', { new_color: value, to_update: left })
+              }
+            />
+          </LabeledList.Item>
+          <LabeledList.Item label="Right Eye">
+            <ColorBox color={eyeColor.right} />{' '}
+            <Button
+              icon="palette"
+              onClick={() => act('pick_color', { to_update: right })}
+              tooltip="Brings up a color pick window to change the light color."
+            />
+            <Button
+              icon="dice"
+              onClick={() => act('random_color', { to_update: right })}
+              tooltip="Randomizes the light color."
+            />
+            <Input
+              value={eyeColor.right}
+              width={6}
+              maxLength={7}
+              onChange={(_, value) =>
+                act('enter_color', { new_color: value, to_update: right })
+              }
+            />
+          </LabeledList.Item>
+        </LabeledList>
+      )}
+    </Section>
+  );
+};
+
+export const HighLuminosityEyesMenu = (props, context) => {
+  const { act, data } = useBackend<HighLuminosityEyesMenuData>(context);
+  const { eyeColor } = data;
+  return (
+    <Window
+      title="High Luminosity Eyes"
+      width={262}
+      height={eyeColor.hasOwner ? (eyeColor.mode ? 259 : 324) : 188}>
+      <Window.Content scrollable>
```

Comment:
i'm not super thrilled on this design, i think it's a bit redundant to use multiple sections and labeled lists which can be squashed into just one.

I'm a hater of scrollable window content, I think one `section fill scrollable` should replace it

Reply:
Aesthetic and vendetta against NT :). Sticky headers just looks entirely better to me, especially for navigational components. I hate having to scroll back to top to navigate elsewhere. Basically, this effect:
![image](https://user-images.githubusercontent.com/42397676/237018856-b51ede4d-8b16-46eb-b6ab-e1af8c18d420.gif)

Reply:
looking at this I think I didn't communicate that window.content was still okay to leave in. i'll message you about this on discord, the scrollable prop goes on section, i meant

Reply:
Here's a version I put together rq that I think embodies minimizing redundant section/labels
![dreamseeker_tnrB1syglL](https://github.com/tgstation/tgstation/assets/42397676/4d9c381c-d64b-489b-9e96-cf7f47a45662)